docs/user_manual_how_to_use_1_operator_mode_additional_functionalities.rst

```diff
@@ -23,8 +23,8 @@ This can be done by first select WOA09 atlas or RTOFS atlas under the *Input dat
 
     The *Input data* button in the *Editor* toolbar.
 
-This will trigger a series of question dialogs that will determine use of the SIS date/position input
-for the query position or an user provided input. After that, a surface sound speed can be applied,
+This will trigger a series of question dialogs for the query position that will either use the SIS data/position
```

Comment:
The dialogs ask about position **, but also** date. SSM does a spatio-temporal query to retrieve the synthetic profile.